robolectric/src/main/java/org/robolectric/internal/dependency/MavenDependencyResolver.java

```diff
@@ -16,7 +16,8 @@
   private final String repositoryId;
 
   public MavenDependencyResolver() {
-    this("https://oss.sonatype.org/content/groups/public/", "sonatype");
+    this(System.getProperty("robolectric.dependency.repo.url", "https://oss.sonatype.org/content/groups/public/"),
```

Comment:
Can you move the `System.getProperty` calls to methods on `RoboSettings`? We're trying to make that a central place to hold all system properties.